packages/babel-parser/test/helpers/fixture-error.js

```diff
@@ -0,0 +1,97 @@
+import { inspect } from "util";
+import Difference from "./difference.js";
+import "./polyfill.js";
+
+const { isArray } = Array;
+const { defineProperty, entries, fromEntries } = Object;
+
+const named = (name, object) => defineProperty(object, "name", { value: name });
+const mapEntries = (object, f) => fromEntries(entries(object).map(f));
+// eslint-disable-next-line no-confusing-arrow
+const toContextError = error =>
+  isArray(error) ? error.map(toContextError) : error.context || error;
+
+export default class FixtureError extends Error {
+  constructor(difference, { cause } = {}) {
+    super();
+
+    // Sigh, still have to manually set the name unfortunately...
+    named(this.constructor.name, this);
+
+    this.difference = difference;
+
+    // Set cause ourselves, since node < 17 has a bug where it won't show it
```

Comment:
It's not a bug, it's just that `Error#cause` reached stage 4 recently :stuck_out_tongue: